static/app/components/events/eventEntries.tsx

```diff
@@ -102,7 +101,6 @@ const EventEntries = ({
           isShare={isShare}
         />
       )}
-      <EventEvidence event={event} group={group} />
```

Comment:
Forgot to remove this in previous refactor - it's rendered in GroupEventDetails now so no need for it here